source/connect-to-mongo/create-a-mongo-client.txt

```diff
@@ -77,9 +77,9 @@ A standard connection string includes the following components:
 
      - Optional. A query string that specifies connection-specific
        options as ``<name>=<value>`` pairs. 
-        ..    TODO: uncomment out when connection options page made
-        ..    See :ref:`java-rs-connection-options` for a full description of
-        ..    these options.
+        
+        See :ref:`java-rs-connection-options` for a full description of
+        these options.
```

Comment:
S: the [style guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear) recommends that these sentences start with the purpose of the link, rather than starting with the link itself:
```suggestion
        For a full description of these options, see the :ref:`java-rs-connection-options` guide.
```